provision/minikube/Taskfile.yaml

```diff
@@ -204,6 +204,13 @@ tasks:
     status:
       - test -f keycloak/providers/keycloak-benchmark-dataset-*.jar
 
+  dataset-import:
+    deps:
+      - gatlinguser
+    cmds:
+      - bash -c "../../dataset/dataset-import.sh {{.CLI_ARGS}}"
```

Comment:
@kami619 Is it safe to use it this way (.CLI_ARGS)? Just curious, because I can see a potential problem in a future for refactoring.

Reply:
I think the alternative would be to parse the full CLI argument? I'd rather not go that route and keep it inside of the dataset import script to parse the parameters, so I'm ok with the solution as it is suggested here.

Reply:
@lhanusov I think its probably safe even considering refactoring, there would be no lift for adapting to new or updated args for the bash script.